primitives/timestamp/src/lib.rs

```diff
@@ -169,12 +169,13 @@ impl TimestampInherentData for InherentData {
 ///
 /// This timestamp is the time since the UNIX epoch.
 #[cfg(feature = "std")]
-fn current_timestamp() -> std::time::Duration {
+pub fn current_timestamp() -> Timestamp {
```

Comment:
Maybe we should as this as `current()` to the `Timestamp` type directly?

Reply:
Done